Modules/Core/Common/test/itkCMakeConfigurationTest.cxx

```diff
@@ -74,7 +75,7 @@ main(int argc, char * argv[])
 {
   if (argc != 2)
   {
-    std::cerr << "Usage: itkCMakeInformationTest <top-of-build-tree>\n";
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " <top-of-build-tree>\n";
```

Comment:
If this does not compile, renaming into `itkCMakeConfigurationTest` should be done.

Reply:
Oh, it is used here:
https://github.com/InsightSoftwareConsortium/ITK/blob/24c3ad91db624592318c886c8d81614e0662ea72/Modules/Core/Common/test/CMakeLists.txt#L557-L560
and it did not get broken. All OK!